controllers/utility.go

```diff
@@ -62,7 +62,7 @@ func outOfOffice(upgradeTime time.Time) bool {
 	if int(upgradeTime.Weekday()) == 6 || int(upgradeTime.Weekday()) == 0 {
 		return true
 	}
-	if upgradeTime.UTC().Hour() <= 7 && upgradeTime.UTC().Hour() >= 16 {
+	if upgradeTime.UTC().Hour() <= 7 || upgradeTime.UTC().Hour() >= 16 {
```

Comment:
Thanks to the test I wrote I saw I made a mistake here 🙈

Reply:
Oops! :slightly_smiling_face: Happens to the best of us.